### DISCUSSION TITLE
Falls Church Priorities

### DISCUSSION THREAD

**Discussion by jkleiber** [team401/2024-Robot-Code#152] [General]
@redPlover @charlesmackenzie In the 2024 Software Roadmap, I have added a page for Falls Church priorities. Before the meeting today (if possible), it would be good if you guys could take a stab at putting the unranked items into the tier list so we can get a rough estimate of how we want to order the work

**Comment by honzikschenk** [comment]
> @redPlover @charlesmackenzie In the 2024 Software Roadmap, I have added a page for Falls Church priorities. Before the meeting today (if possible), it would be good if you guys could take a stab at putting the unranked items into the tier list so we can get a rough estimate of how we want to order the work

@jkleiber I ranked the tasks based on what I think would be most useful and take less time. Do you have any suggestions?

**Reply by jkleiber** [reply]
Looks good, I left a couple comments on lower priority things
